### PR TITLE
Profile for v1.0

### LINKED ISSUE #11: As a user, I would like to set my height, weight and net calorie goal
so that I can start tracking my progress.

### LINKED ISSUE #14: As a user, I would like to calculate my BMI based on my height and weight in the profile
so that I am able to track my fitness progress quantitively.

### PR DESCRIPTION
**Pull request by weidak**
Profile.java created for the storing of height, weight, net goal calories and calculate BMI (for now)

Fixes #11 
Fixes #14

### CODE REVIEW

**Review by tttyyzzz — COMMENTED** [review]
seems good to me, codes are short and readable such that no comments are needed

File: src/main/java/seedu/duke/Profile/Profile.java [hunk 1]
```diff
@@ -1,2 +1,48 @@
-package seedu.duke.Profile;public class Profile {
+package seedu.duke.Profile;
+
+public class Profile {
```
Comment (tryyang2001) [inline src/main/java/seedu/duke/Profile/Profile.java]: was thinking if we need to store the name of the user in the future, but looks good for now

Comment (weidak) [reply src/main/java/seedu/duke/Profile/Profile.java]: Same, i not sure if we should be creating multiple profiles for multiple users. potential expansion?

File: src/main/java/seedu/duke/profile/Profile.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        this.calorieGoal = calorieGoal;
+    }
+
+    public double calculateBmi() {
+        double heightInM = ((double) height) / 100.0;
+        double bmi = Math.round((weight / (Math.pow(heightInM, 2))) * 10) / 10.0;
+        return bmi;
+    }
+
+    public static double calculateBmi(int height, int weight) {
```
Comment (tlyi) [inline src/main/java/seedu/duke/profile/Profile.java:38]: I think that we could use a `double` type for `height` and `weight` instead to allow for more precise numbers!

File: src/main/java/seedu/duke/profile/Profile.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
+package seedu.duke.profile;
+
+public class Profile {
```
Comment (tlyi) [inline src/main/java/seedu/duke/profile/Profile.java:3]: Remember to add JavaDoc for this class!

File: src/test/java/seedu/duke/profile/ProfileTest.java [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    void calculateBmi_negativeInputs_expectException() {
+        assertThrows(InvalidCharacteristicException.class,
+            () -> Profile.calculateBmi(-171.2, -59.8));
+    }
+
+    @Test
+    void retrieveBmiStatus_healthyBmiInput_expectHealthyStatus() {
+        assertEquals("Healthy", Profile.retrieveBmiStatus(22.5));
+    }
+}
```
Comment (tttyyzzz) [inline src/test/java/seedu/duke/profile/ProfileTest.java:49]: Perhaps you could try when height equals to 0.

Comment (weidak) [reply src/test/java/seedu/duke/profile/ProfileTest.java:76]: Good point, missed that out.

**Pull request merged**